swtbot_test/src/parameters/ProjectParameters.java

```diff
@@ -353,12 +353,12 @@ public class AmazonConfig{
 	}
 	
 	public class FileLocation{
-		public static final String AZURE_RTOS_LOCATION = "D:\\rtos_package\\Azure\\6.2.1-rx-2.0.0";
-		public static final String FIT_MODULES_LOCATION = "D:\\FITModules";
+		public static final String AZURE_RTOS_LOCATION = "D:\\rtos_package\\Azure_Final_NoChange";
+		public static final String FIT_MODULES_LOCATION = "D:\\RDP1.39";
```

Comment:
E: this location is your hardcode or legit FIT module location?

Reply:
These locations have to be changed depends on users machine. This is legit Fit Modules location on my machine

Reply:
No, you must define a common location where the package is default downloaded.
These locations have to be changed depends on users machine. --> should not do this